packages/typespec-ts/test/modularIntegration/versioningRemoved.spec.ts

```diff
@@ -28,30 +20,4 @@ describe("VersioningRemoved Rest Client", () => {
     assert.strictEqual(result.enumProp, "enumMemberV2");
     assert.strictEqual(result.unionProp, "bar");
   });
-
-  it("versioning removed test modelV3", async () => {
```

Comment:
why these tests are removed ?

Reply:
refer to: https://github.com/microsoft/typespec/blob/main/packages/http-specs/specs/versioning/removed/mockapi.ts